### PR TITLE
Add the more_than plugin to list of exernal extensions

### PR DESCRIPTION
**Pull request by eriklovmo**
Hi Jeremy, 

Thanks for all your hard work on Sequel!

I wrote a small [plugin](https://github.com/eriklovmo/sequel_more_than) to decide whether a dataset's row count is less than, less than or equal to, greater than, or greater than or equal to a given number. This pull request adds it to the Sequel website's list of external extensions.

### CODE REVIEW

File: www/pages/plugins.html.erb [hunk 1]
```diff
@@ -1125,6 +1125,9 @@
 <a class="a" href="http://github.com/jsvd/sequel_vectorized">sequel_vectorized </a>
 <span class="ul__span">Allows Sequel::Dataset to be exported as an Hash of Arrays and NArrays.</span>
 </li>
+<li class="ul__li ul__li--grid">
+<a class="a" href="https://github.com/eriklovmo/sequel_more_than">sequel_more_than </a>
+<span class="ul__span">Efficiently compare a Sequel::Dataset's row count against a given number.</span> </li>
```
Comment (jeremyevans) [inline www/pages/plugins.html.erb:1130]:
```suggestion
<span class="ul__span">Efficiently compare a Sequel::Dataset's row count against a given number.</span>
</li>
```

**Review by jeremyevans — CHANGES_REQUESTED** [review]
Looks good.  I would just like to make one suggested change to keep the formatting in the file consistent.

**Comment by eriklovmo** [comment]
> Looks good. I would just like to make one suggested change to keep the formatting in the file consistent.

Thanks, fixed in c0a5b0af6cfc761ae9c7ec0f9a2435aecc49c09b.

**Comment by jeremyevans** [comment]
Thanks for the patch!

**Pull request merged**